tutorials/Spearmans_Correlation.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
This tutorial is mostly a duplication of the Persons correlation tutorial.

I suggest setting this up to be more focused on the Spearmans correlation coef.

Rather than just using the same synthetic forecast and observation data, it would be far more useful if you generated synthetic time series with non-linear relationships, rather than linear relationships; and then constructed a story around that data.

You could then at the end say/show that pearsons == Spearmans when the relationship is linear.